src/api/app/controllers/staging/staging_projects_controller.rb

```diff
@@ -52,6 +52,14 @@ def accept
 
     # check general state
     raise StagingProjectNotAcceptable, 'Staging project is not in state acceptable.' unless can_accept?
+
+    # Disabling build for all repositories and architectures.
+    build_flag = @staging_project.flags.find_or_initialize_by(flag: 'build', repo: nil, architecture_id: nil)
+    build_flag.update(status: 'disable')
+
+    # Remove all the build flags enabled by the user.
+    @staging_project.flags.where(flag: 'build', status: 'enable').destroy_all
```

Comment:
:thought_balloon: This could be a `StagingProject#clear_flags!`

Reply:
remembere that StagingProject is just a concern included into all projects.

Reply:
We've been talking offline and we've decided to not apply this suggestion.